packages/select/test/scrollable-viewport.test.js

```diff
@@ -74,35 +74,42 @@ describe('scrollable viewport', () => {
     };
   });
 
-  it('should toggle bottom-aligned attribute depending on the part of the viewport', () => {
-    enterKeyDown(valueButton);
-    expect(overlay.hasAttribute('bottom-aligned')).to.be.true;
-
-    scrollContainer(scrollableContainer, 150);
-    expect(overlay.hasAttribute('bottom-aligned')).to.be.false;
-  });
-
   it('should update the position on scrolling', () => {
     enterKeyDown(valueButton);
 
-    expect(overlay.getBoundingClientRect().bottom).to.be.closeTo(inputFieldBlock.getBoundingClientRect().bottom, 1);
-    expect(overlay.getBoundingClientRect().left).to.be.closeTo(inputFieldBlock.getBoundingClientRect().left, 1);
+    expect(overlay.$.overlay.getBoundingClientRect().top).to.be.closeTo(inputFieldBlock.getBoundingClientRect().top, 1);
```

Comment:
Seems that the threshold for switching between top/bottom -align is different in `vaadin-overlay-position-mixin` so needed to change `bottom`/`top` in some assertions.